sdk/cosmosdb/cosmos/src/client/Offer/Offer.ts

```diff
@@ -40,7 +40,13 @@ export class Offer {
       resourceId: this.id,
       options,
     });
-    return new OfferResponse(response.result, response.headers, response.code, this);
+    return new OfferResponse(
```

Comment:
Offer need to have diagnostics?

Reply:
Yeah, Offer is an api just like anything else. From a API perspective Offer and Item are not different.